testfiles/test1.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
A couple of test files for convenience. Can remove if unnecessary.

Reply:
makes sense to me to have basic fixtures like this!